gradle.properties

```diff
@@ -36,9 +36,9 @@ jacocoVersion=0.8.2
 hamcrestVersion=2.2
 
 # Version and base tags can be overridden at build time.
-connectorVersion=0.10.0-SNAPSHOT
-pravegaVersion=0.10.0-2976.ea3db6f-SNAPSHOT
-schemaRegistryVersion=0.3.0-74.dee4154-SNAPSHOT
+connectorVersion=0.10.2-SNAPSHOT
+pravegaVersion=0.10.2-2996.f1d4853-SNAPSHOT
+schemaRegistryVersion=0.4.0-78.f1b3eef-SNAPSHOT
```

Comment:
We change the schema registry version to `0.3.0` here, as 0.4.0 aligns with Pravega 0.11 which may have some conflict.

Reply:
fixed in [060d538](https://github.com/pravega/flink-connectors/pull/614/commits/060d53867e721d78fa5f778effd8efc5cbe8bcfe)